src/app/examples/pos/pos.html

```diff
@@ -24,8 +24,7 @@
 <div
   *ngIf="
     (service.meow$ | async | isSuccess) ||
-    ((service.meow$ | async | isInProgress) &&
-      (service.meow$ | async | inProgressValue))
+    (service.meow$ | async | isInProgress)
```

Comment:
I'm just briefly checking from my phone, I have not run the example. My understanding is that we don't need to show this element when it's in progress, since we don't have a cat anymore

Reply:
You are right 👍